.devcontainer/devcontainer.json

```diff
@@ -1,79 +1,43 @@
-// For format details, see https://aka.ms/devcontainer.json. For config options,
-// see documentation located at the following URL:
-// https://github.com/OpenINF/openinf-docker-fish/tree/main/library-scripts/docs
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/docker-outside-of-docker
 {
-  "name": "Jekyll (OpenINF Community)",
-  "build": {
-    "dockerfile": "Dockerfile"
-  },
-  "runArgs": ["--cap-add=SYS_ADMIN", "--security-opt", "seccomp=unconfined"],
-  // Configure tool-specific properties.
-  "customizations": {
-    // Configure properties specific to VS Code.
-    "vscode": {
-      // Set *default* container-specific settings.json values on container
-      // create.
-      "settings": {
-        "terminal.integrated.profiles.linux": {
-          "fish": {
-            "path": "fish"
-          }
-        },
-        "terminal.integrated.defaultProfile.linux": "fish"
-      },
+	"name": "Docker outside of Docker",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/base:bullseye",
 
-      // Add the IDs of extensions you want installed when the container is created.
-      "extensions": [
-        // https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments
-        "aaron-bond.better-comments",
-        // https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint
-        "davidanson.vscode-markdownlint",
-        // https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint
-        "dbaeumer.vscode-eslint",
-        // https://marketplace.visualstudio.com/items?itemName=dprint.dprint
-        "dprint.dprint",
```

Comment:
finicky extension doesn't know where to look for the CLI of `dprint` command on host machine (meaning the Linux VM)